tests/www/employee_record_views/test_create.py

```diff
@@ -284,11 +285,41 @@ def test_job_seeker_without_address(self):
         side_effect=mock_get_geocoding_data,
     )
     def test_job_seeker_address_geolocated(self, _mock):
+        job_seeker = JobSeekerWithAddressFactory(
+            for_snapshot=True,
+            with_mocked_address=BAN_GEOCODING_API_RESULTS_FOR_SNAPSHOT_MOCK,
+        )
+        job_application = JobApplicationWithApprovalNotCancellableFactory(
+            to_company=self.company,
+            job_seeker=job_seeker,
+        )
+
         # Accept geolocated address provided by mock and pass to step 3
-        response = self.client.get(self.url)
-        url = reverse("employee_record_views:create_step_3", args=(self.job_application.pk,))
-        response = self.client.get(url)
+        response = self.client.get(reverse("employee_record_views:create_step_2", args=(job_application.pk,)))
+        form_soup = parse_response_to_soup(
+            response,
+            selector=".s-section form",
+            replace_in_attr=[
+                (
+                    "action",
+                    reverse("employee_record_views:create_step_2", args=(job_application.pk,)),
+                    "/employee_record/create_step_2/[PK of JobDescription]",
+                ),
+                (
+                    "href",
+                    reverse("employee_record_views:create_step_3", args=(job_application.pk,)),
+                    "/employee_record/create_step_3/[PK of JobDescription]",
+                ),  # Go to next step button
+                (
+                    "href",
+                    reverse("employee_record_views:create", args=(job_application.pk,)),
+                    "/employee_record/create/[PK of JobDescription]",
+                ),  # Go to previous step button
```

Comment:
A priori ce sont des `PK of JobApplication` :grimacing:

Reply:
:see_no_evil: